arch/arm/Kconfig

```diff
@@ -6,9 +6,16 @@
 menu "ARM Options"
 	depends on ARM
 
-source "arch/arm/core/Kconfig"
+source "arch/arm/core/aarch32/Kconfig"
+source "arch/arm/core/aarch64/Kconfig"
```

Comment:
rsource

Reply:
ACK. Fix coming in V3. Closing to declutter the story. Thanks.